code/modules/surgery/tools.dm

```diff
@@ -141,7 +139,7 @@
 	righthand_file = 'icons/mob/inhands/equipment/medical_righthand.dmi'
 	item_state = "scalpel"
 	flags_1 = CONDUCT_1
-	force = 10
+	force = 7
```

Comment:
For reference 7 damage is the amount of damage caused by a punch. Toolboxes have 12 damage and welders have 15.